scripts/dy_services_helpers/platform_initialiser.py

```diff
@@ -36,6 +37,14 @@ def __init__(self):
 @tenacity.retry(wait=tenacity.wait_fixed(2), stop=tenacity.stop_after_attempt(5) | tenacity.stop_after_delay(20))
 def init_db():
     db = DbSettings()    
+    _metadata = sa.MetaData()

```

Comment:
mmhhh same issue as always, should we put all metadata in a single shared place or copy&paste it around? Anoghter solution would be to isolate uservices databases,i.e. one db per uservice. This way each uservice might control its own tables

Reply:
it's probably more maintainable if each uservice takes care of its stuff somehow.